pkgs/development/python-modules/qcodes-loop/default.nix

```diff
@@ -0,0 +1,82 @@
+{ lib
+, fetchPypi
+, pythonOlder
+, buildPythonPackage
+, qcodes
+, h5py
+, lazy-loader
+, matplotlib
+, numpy
+, pandas
+, versioningit
+, xarray
+, hickle
+, ipython
+, slack-sdk
+, hypothesis
+, pytest-xdist
+, pytest-mock
+, pyqtgraph
+, pyqt5
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "qcodes-loop";
+  version = "0.1.1";
+
+  disabled = pythonOlder "3.8";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "qcodes_loop";
+    sha256 = "sha256-pDR0Ws8cYQifftdE9dKcSzMxmouFo4tJmQvNanm6zyM=";
+  };
+
+  propagatedBuildInputs = [
+    qcodes
+    h5py
+    lazy-loader
+    matplotlib
+    numpy
+    pandas
+    versioningit
```

Comment:
Is this a runtime dependency? We don't have a git directory so it could be none functional